chapters/en-UK/xctest/types_of_testing.md

```diff
@@ -1,21 +1,23 @@
 # Types of Testing
 
-A test is a way of verifying code, and making your assumptions explicit. When you make a change somewhere, tests exist to show the implicit connections between objects.
```

Comment:
How do tests reveal implicit connections? Do you mean they'll fail when you change some algorithm somewhere?